tests/memory_tests/test_runner/conftest.py

```diff
@@ -0,0 +1,547 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""
+Basic high-level plugin file for pytest.
+
+See [Writing plugins](https://docs.pytest.org/en/latest/writing_plugins.html)
+for more information.
+
+This plugin adds the following command-line options:
+
+* `--test_conf` - Path to test configuration file. Used to parametrize tests.
+  Format: YAML file.
+* `--exe` - Path to a test binary to execute.
+* `--niter` - Number of times to run executable.
+"""
+
+import hashlib
+import json
+import logging
+# pylint:disable=import-error
+import os
+import sys
+from copy import deepcopy
+from inspect import getsourcefile
+from pathlib import Path
+
+import pytest
+import yaml
+from jsonschema import validate, ValidationError
+
+UTILS_DIR = os.path.join(Path(__file__).parent.parent.parent, "utils")
+sys.path.insert(0, str(UTILS_DIR))
+
+from plugins.conftest import *
+from path_utils import check_positive_int
+from proc_utils import cmd_exec
+from platform_utils import get_os_name, get_os_version, get_cpu_info
+from utils import metadata_from_manifest, DATABASES, DB_COLLECTIONS
+
+MEMORY_TESTS_DIR = os.path.dirname(os.path.dirname(__file__))
+sys.path.append(MEMORY_TESTS_DIR)
+
+from test_runner.utils import query_memory_timeline, REFS_FACTOR
+
+
+OMZ_NUM_ATTEMPTS = 1
```

Comment:
it should be greater to decrease number of problems related to network connection - refer to `get_testdata.py`